dev-docs/analytics/asteriobid.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: analytics
+title: AsterioBid
+description: AsterioBid Analytics Adapter
+modulecode: asteriobid
+---
+
+#### Registration
```

Comment:
Is there no example configuration you want to add? A lot of analytics adapters have at least

```js
    pbjs.enableAnalytics({
        provider: 'asteriobid'
    });
```